exporter/jaegerexporter/exporter.go

```diff
@@ -32,7 +33,7 @@ import (
 // newTraceExporter returns a new Jaeger gRPC exporter.
 // The exporter name is the name to be used in the observability of the exporter.
 // The collectorEndpoint should be of the form "hostname:14250" (a gRPC target).
-func newTraceExporter(cfg *Config) (component.TraceExporter, error) {
+func newTraceExporter(logger *zap.Logger, cfg *Config) (component.TraceExporter, error) {
```

Comment:
Nit: use config, logger - consistent with other places :)

Reply:
Interesting, I thought Go's conventions would suggest that logger/context should be provided before any "local" parameter.

Reply:
I know about context, but if that is the case for logger as well then... We don't follow it :))